b2share/modules/schemas/serializers.py

```diff
@@ -60,7 +70,24 @@ def block_schema_version_to_dict(block_schema_version):
 
 def block_schema_version_to_json_serializer(block_schema_version, code=200,
                                             headers=None):
-    response = jsonify(block_schema_version_to_dict(block_schema_version))
+    """Serializes block schema version to json response.
+
+    Args:
+        block_schema_version
+            (:class:`b2share.modules.schemas.api:BlockSchemaVersion`):
+            block schema version that will be serialized.
+
+        code: http response status.
+
+        headers: additional http response headers.
+
+    Returns:
+        response: serialized response from BlockSchemaVersion.
+    """
+    if block_schema_version is not None:
+        response = jsonify(block_schema_version_to_dict(block_schema_version))
+    else:
+        response = jsonify({})
```

Comment:
What is the scenario in which the `block_schema_version` is `None` and it is not an error?